language-bindings/rust/build.rs

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2023 Liquid Reply GmbH. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+extern crate bindgen;
+extern crate cmake;
+
+use cmake::Config;
+use std::{env, path::PathBuf};
+
+fn main() {
+    let mut config = Config::new("../..");
+    let dst = config
+        .generator("Unix Makefiles")
+        .define("WAMR_BUILD_TARGET", env::var("TARGET").unwrap().split("-").next().unwrap().to_uppercase())
```

Comment:
I think we'd better let CMakeFiles to decide the value of `WAMR_BUILD_TAEGET`.

Plus, I am not sure `-G 'Unix Makefiles'` is suitable here. at least, as I know, it won't work on Windows(not with mingw)

Reply:
RE build target: this is a leftover from the crate I forked, I had not tried to remove it.
RE generator: I just have no experience with this, I haven't used windows or mingw in years.

The build process seems to work fine without these two configs, so I remove them.